algebird-core/src/main/scala/com/twitter/algebird/QTree.scala

```diff
@@ -258,6 +289,7 @@ case class QTree[A](
   }
 
   def interQuartileMean(implicit n: Numeric[A]): (Double, Double) = {
+    implicit val monoid: Monoid[A] = Ring.numericRing[A]
```

Comment:
@ianoc just to be sure I understand: you like the solution of taking both Monoid and Numeric?